recipe/meta.yaml

```
@@ -1,58 +1,60 @@
-{% set version = "0.17.8" %}
+{% set version = "0.17.11" %}
 
 package:
   name: great-expectations
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/g/great-expectations/great_expectations-{{ version }}.tar.gz
-  sha256: a13bb1827f3c710d059db18cea727961b8fb90d1f7656d03e1c9c5bcf7bb52bf
+  sha256: 484106cd2cd5e2ad19a13ced3283eafa6855822b2bbfed0df545674e6cfac6ea
 
 build:
-  noarch: python
+  skip: True  # [py<36]
```

```suggestion
  skip: True  # [py<38]
```
https://github.com/great-expectations/great_expectations/blob/0d308ac699bc9453b2c3ca92bf59a04ed4450113/setup.py#L117